docs-src/docs/reference/setup-cli.md

```diff
@@ -1,278 +1,212 @@
-  </div>
-  <div class="divRow">
-    <div class="divCellNoWrap">--secret\_key=_awsSecretAccessKeyID_</div>
-    <div class="divCell">The Secret Access Key ID of an IAM user associated with the AWS stack in which you are creating the Layer0 instance.</div>
-  </div>
-  <div class="divRow">
-    <div class="divCellNoWrap">--region=_awsRegion_</div>
-    <div class="divCell">The AWS region in which the Layer0 instance resides.</div>
-  </div>
-  <div class="divRow">
-    <div class="divCellNoWrap">--dockercfg=_filepath_</div>
-    <div class="divCell">The path to a valid Docker configuration file (likely found at `~/.docker/config.json` for Docker versions 1.7 or greater, or `~/.dockercfg` for versions less than 1.7).</div>
-  </div>
-  <div class="divRow">
-    <div class="divCellNoWrap">--vpc=_vpcID_</div>
-    <div class="divCell">The ID of a VPC. If blank, **l0-setup** will create a new VPC.</div>
-  </div>
-</div>
+```
+$ l0-setup init [options] <instance_name> 
+```
+
+### Options
+* `--docker-path` - Path to docker config.json file. 
+This is used to include private Docker Registry authentication for this Layer0 instance.
+* `--module-source` - The source input variable is the path to the Terraform Layer0. 
+By default, this points to the Layer0 github repository. 
+Using values other than the default may result in undesired consequences.
+* `--version` - The version input variable specifies the tag to use for the Layer0
+Docker images: `quintilesims/l0-api` and `quintilesims/l0-runner`.
+* `--aws-access-key` - The access_key input variable is used to provision the AWS resources
+required for Layer0. 
+This corresponds to the Access Key ID portion of an AWS Access Key.
+It is recommended this key has the `AdministratorAccess` policy. 
+* `--aws-secret-key` The secret_key input variable is used to provision the AWS resources
+required for Layer0. 
+This corresponds to the Secret Access Key portion of an AWS Access Key.
+It is recommended this key has the `AdministratorAccess` policy.
+* `--aws-region` - The region input variable specifies which region to provision the
+AWS resources required for Layer0. The following regions can be used:
+    - us-west-1
+    - us-west-2
+    - us-east-1
+    - eu-west-1
+
+
+* `--aws-ssh-key-pair` - The ssh_key_pair input variable specifies the name of the
+ssh key pair to include in EC2 instances provisioned by Layer0. 
+This key pair must already exist in the AWS account. 
+The names of existing key pairs can be found in the EC2 dashboard.
 
 ---
-
-##Backup
-The **backup** command is used to back up your Layer0 configuration files to an S3 bucket. This command is most often used when migrating between versions of Layer0. The **backup** command also runs automatically every time you execute the **l0-setup apply** command.
+## Plan
+The **plan** command is used to show the planned operation(s) to run during the next `apply` on a Layer0 instance
```

Comment:
[nit] just an opinion so no change required; i feel like you would need to elaborate on this. It isn't clear why I would use plan.